modules/examiner/ajax/editCertificate.php

```diff
@@ -0,0 +1,169 @@
+<?php
```

Comment:
We shouldn't be adding new ajax/ scripts, we should be getting rid of them. I think this should be converted to an NDB_Page which overrides the process function to remove the middleware and uses the rest of the LORIS framework for hasAccess/setup/etc.

Reply:
> We shouldn't be adding new ajax/ scripts, we should be getting rid of them.

@driusan Could you clarify this? I didn't know we had this policy.